js/rank_states_utils.js

```diff
@@ -83,6 +84,7 @@ function rankEm(barData) {
   openBars
     .style('fill', "rgba(255, 255, 255, 0.0)")
     .style('stroke',"rgb(198,198,198)")
+    .style('pointer-events', 'fill')
```

Comment:
It is still doing the blinking thing, but this is minor, so I don't think we should spend much more time fixing it.

Reply:
Not blinking -- it's because of the bug that I introduced with being able to cheat and hover over the open bar names and see the answer.